### PR TITLE
BXMSDOC-8680 Add note about native builds and containerfile

### PR DESCRIPTION
**Pull request by emmurphy1**

### CODE REVIEW

File: doc-content/enterprise-only/getting-started-kogito/proc-kogito-microservices-deploying-on-ocp-console-custom-image-build.adoc [hunk 1]
```diff
@@ -7,6 +7,19 @@ The operator uses the following custom resources to deploy domain-specific micro
...
 * `KogitoRuntime` starts the runtime image and configures it as per your requirements.
 
+[NOTE]
+====
+The {PRODUCT} builder image does not supports native builds.  However, you can perform a custom build and use `Containerfile` to build the container image as shown in the following example:
+
+`FROM registry.redhat.io/rhpam-7-tech-preview/rhpam-kogito-runtime-native-rhel8:{ENTERPRISE_VERSION_LONG}`
+
+`ENV RUNTIME_TYPE quarkus`
+
```
Comment (spolti) [inline doc-content/enterprise-only/getting-started-kogito/proc-kogito-microservices-deploying-on-ocp-console-custom-image-build.adoc:17]: Seems that the COPY command is missing.

something like:

```
# To build the native binary with Mandrel please visit <doc to Quarkus native builds>
COPY --chown=1001:root target/*-runner $KOGITO_HOME/bin
```

**Comment by radtriste** [comment]
Apart from comment from @spolti, change lgtm

File: doc-content/enterprise-only/getting-started-kogito/proc-kogito-microservices-deploying-on-ocp-console-custom-image-build.adoc [hunk 2]
```diff
@@ -7,6 +7,19 @@ The operator uses the following custom resources to deploy domain-specific micro
...
+This feature is Technology Preview only.
+
+To build the native binary with Mandrel, see https://https://access.redhat.com/documentation/en-us/red_hat_build_of_quarkus/rhbq-documentation-2-13/guide/8acbc1ec-17d6-4e5c-9651-6e2c4df33f8a[Compiling your Quarkus applications to native executables].
```
Comment (radtriste) [inline doc-content/enterprise-only/getting-started-kogito/proc-kogito-microservices-deploying-on-ocp-console-custom-image-build.adoc]:
```suggestion
To build the native binary with Mandrel, see https://access.redhat.com/documentation/en-us/red_hat_build_of_quarkus/rhbq-documentation-2-13/guide/8acbc1ec-17d6-4e5c-9651-6e2c4df33f8a[Compiling your Quarkus applications to native executables].
```

File: doc-content/enterprise-only/getting-started-kogito/proc-kogito-microservices-deploying-on-ocp-console-custom-image-build.adoc (hunk 1 above)
Comment (emmurphy1) [reply doc-content/enterprise-only/getting-started-kogito/proc-kogito-microservices-deploying-on-ocp-console-custom-image-build.adoc:17]: @radtriste @spolti So to me it looks like the commented line is referring to the doc and then there is the command. I added the command with the other commands, then the doc after.Is this right?

Comment (radtriste) [reply doc-content/enterprise-only/getting-started-kogito/proc-kogito-microservices-deploying-on-ocp-console-custom-image-build.adoc:17]: I am ok with that

**Review by radtriste — APPROVED** [review]

**Pull request merged**